Jenkins/Jenkinsfile

```diff
@@ -0,0 +1,67 @@
+pipeline {
+	agent any
+
+    tools {
+        // Install the Maven version configured in Global Tool Configuration
+        maven 'Maven 3.8.4'
+        // Install the JDK version configured in Global Tool Configuration
+        jdk 'JDK 11'
+        // Install the Node version configured in Global Tool Configuration
+        nodejs "NodeJS 20.10"
+    }
+
+	environment {
+		// Replace with your namespace
+		ENDOR_API = "https://api.endorlabs.com"
+        ENDOR_NAMESPACE = "jenkins-demo"
+		// Move generated ENV details into endorctl config vars
+		ENDOR_API_CREDENTIALS_KEY = "$ENDOR_API_CREDS_USR"
+		ENDOR_API_CREDENTIALS_SECRET = "$ENDOR_API_CREDS_PSW"
+	}
+
+	stages {
+		stage ('OWASPJavaBenchmark - Checkout') {
+			steps {
+				checkout([$class: 'GitSCM', branches: [[name: '*/master']], doGenerateSubmoduleConfigurations: false, extensions: [], submoduleCfg: [], userRemoteConfigs: [[credentialsId: '', url: 'https://github.com/OWASP-Benchmark/BenchmarkJava.git']]]) 
+			}
+		}
+
+		stage ('OWASPJavaBenchmark - Build') {
+			steps {
+				// Build the package
+				sh """ 
+				mvn clean package 
+				"""
+			}
+		}
+	
+		stage ('OWASPJavaBenchmark - ci_run Scan') {
+			steps {
+				// Test npm and node installations
+                sh 'npm --version'
+                sh 'node --version'
+
+                // Define the Node.js installation name configured in Jenkins
+                script {
+                    NODEJS_HOME = tool name: 'NodeJS', type: 'jenkins.plugins.nodejs.tools.NodeJSInstallation'
+                    PATH = "$NODEJS_HOME/bin:${env.PATH}"
+                }
+
+                // Download and install endorctl.
+                sh 'npm install -g endorctl'
```

Comment:
I'm thinking we might prefer not to require `npm` to install `endorctl` in the pipeline.  can we replace with similar approach as the other examples to download the latest release and verify the checksum?  Then we remove the dependency on another package to be present and is consistent with the other examples

Reply:
I agree, I will make required changes.

Reply:
@scott-es I have updated Jenkinsfile to use endorctl executable instead of npm package.